tests/integration/llm/sagemaker-endpoint-tests.py

```diff
@@ -256,13 +264,21 @@ def mme_test(name, image_type):
 
         predictor = mme.deploy(
             1,
-            config.get("instance_type", DEFAULT_INSTANCE_TYPE),
+            instance_type,
             serializer=sagemaker.serializers.JSONSerializer(),
             deserializer=sagemaker.deserializers.JSONDeserializer())
-        for model in list(mme.list_models()):
+        for i, model in enumerate(list(mme.list_models())):
             outputs = predictor.predict(DEFAULT_PAYLOAD, target_model=model)
             print(outputs)
 
+            if run_benchmark:
```

Comment:
do we convert `run_benchmark` to a bool anywhere? this would run if the value is "true" or "false", right (string values)?

Reply:
Good one, missed it. Handled it now.